mne_bids/mne_bids.py

```diff
@@ -72,10 +74,19 @@ def _channels_tsv(raw, fname, verbose):
         The data as MNE-Python Raw object.
     fname : str
         Filename to save the channels.tsv to.
+    overwrite : str, one of ('overwrite', 'error', 'clear')
+        How the file should be handled if is exists already.
+        Defaults to 'overwrite'.
+        If overwrite = `overwrite` or `clear` the old file will be removed and
+        replaced with a new one.
```

Comment:
It's not clear from the docstring what's the difference between "overwrite" and "clear" ...

Reply:
so you propose to process the `overwrite` argument before passing to this function?
Ie.
`'overwrite' or 'clear' -> 'overwrite'`
`'error'` stays as `'error'`
(just making sure I understand you correctly)

Reply:
my worry (?) with having 3 arguments for each private function is that it a) makes it a bit more cluttered, and b) I think it will cause the code to look a bit... weird.
What I mean is that if you took the `_events_tsv` function for example, `overwrite` and `clear` should both do the exact same thing to this file.
You would also have a check for each argument in a dependent way, meaning you would get code that looked like:
```python
if append or overwrite:
    # do something
elif clear:
    # do something else
else:
    raise OSError
```
It doesn't look *that* awful, but it feels kind of wrong, and also is sort of against the whole
> There should be one-- and preferably only one --obvious way to do it.